WalletWasabi.Daemon/ArgumentHelpers.cs

```diff
@@ -6,25 +6,30 @@ namespace WalletWasabi.Daemon;
 
 public static class ArgumentHelpers
 {
-	public static bool TryGetValue<T>(string key, string[] args, Func<string, T> converter, [NotNullWhen(true)] out T? value)
+	public static bool TryGetValue(string key, string[] args, [NotNullWhen(true)] out string? value)
```

Comment:
It's OK that these two methods are not generic anymore, bc we only use them with `string`s, right?

Reply:
Nice, just wanted to be sure.